axlearn/experiments/text/gpt/fuji.py

```diff
@@ -363,7 +364,7 @@ def model_config(
         hidden_dim=hidden_dim,
         num_heads=num_heads,
         vocab_size=vocab_size,
-        stack_cfg=stack_cfg if stack_cfg is not None else RepeatedTransformerLayer.default_config(),
+        stack_cfg=stack_cfg if stack_cfg is not None else StackedTransformerLayer.default_config(),
```

Comment:
does weight only offload work with RepeatedTransformerLayer? @hanzhi713

Reply:
it works, using StackedTransformerLayer just takes a lot of time to compile. So for 405b it wasn't feasible to use it.